src/hooks/useDeleteVideo.tsx

```diff
@@ -11,7 +11,7 @@ export const useDeleteVideo = () => {
   const { joystream } = useJoystream()
   const handleTransaction = useTransaction()
   const { activeMemberId } = useAuthorizedUser()
-  const removeAssetsFromUploads = useUploadsStore((state) => state.removeAssetsWithParent)
+  const { removeAssetsWithParent: removeAssetsFromUploads } = useUploadsStore((state) => state.actions)
```

Comment:
If you need a specific action, it's better to do:
```suggestion
  const removeAssetsFromUploads = useUploadsStore((state) => state.actions.removeAssetsWithParent)
```